src/engine/matchers/cont_all.js

```diff
@@ -19,25 +19,18 @@ limitations under the License.
 
 const log = require('debug')('splitio-engine:matcher');
 const intersection = require('lodash/intersection');
-const isArray = require('lodash/isArray');
 
-function containsAllMatcherContext(vo /*: whitelistObject */) /*: Function */ {
+function containsAllMatcherContext(splitValue /*: array */) /*: Function */ {
```

Comment:
I've went for ruleAttr (the value set on the editor) and runtimeAttr (the value we receive from the dev).
We should get an agreement on a terminology for those two values, so it's easier to just read the code and understand which value is the one you're reading.

We can discuss wether it should be ruleAttr or ruleValue or ruleString/ruleSet/ruleNumber (and apply the same to runtimeAttr) because it would be an easy replace.